package.json

```diff
@@ -11,7 +11,7 @@
     "start-client-example": "npm run start --workspace=ifc-example-client",
     "test": "npm test --workspaces --if-present",
     "commit": "git-cz",
-    "lint": "commitlint -f 321fe10d3ca4a45b91a3693d76412bb0aa796e32",
+    "lint": "commitlint --from c6ce5b2c3ad19bf49b8fb340dda4cb526cb39708",
```

Comment:
`--f` and `--from` is the same I just prefer `--from` because it differentiates it from "force"

Reply:
Seems the merge strategy ate the other commit. Oh well.